### PR TITLE
chore: add eslint rules

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
Added two rules to make the code more beautiful.

### CODE REVIEW

File: parser.js [hunk 1]
```diff
@@ -981,11 +981,11 @@
         // ControlEscape
         var codePoint = 0;
         switch (res[0]) {
-          case 't': codePoint = 0x009; break;
-          case 'n': codePoint = 0x00A; break;
-          case 'v': codePoint = 0x00B; break;
-          case 'f': codePoint = 0x00C; break;
-          case 'r': codePoint = 0x00D; break;
+        case 't': codePoint = 0x009; break;
```
Comment (nicolo-ribaudo) [inline parser.js]: This is odd, `case` should be more indented than `switch`.

Comment (jviereck) [reply parser.js]: @liuxingbaoyu , can you do something about this one? I agree with @nicolo-ribaudo that the indention is odd - the `case` should be more indented than the `switch`.

Comment (liuxingbaoyu) [reply parser.js]: It has been fixed!

**Comment by jviereck** [comment]
Thanks for your contribution!

**Pull request merged**